.github/pull_request_template.md

```diff
@@ -16,12 +16,12 @@ Fixes #ISSUE-NUMBER (eg. Fixes #147)
 
 Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
 
-- [ ] Test A
-- [ ] Test B
+- Test A
```

Comment:
We could just remove these I think. People can list however they tested their PR

Reply:
removed!